integration_test/third_party_apps_data/applications/iis/metadata.yaml

```diff
@@ -38,6 +39,73 @@ expected_metrics:
   labels:
     http_method: .*
   representative: true
+
+# v2 metrics
+- type: workload.googleapis.com/iis.request.count
```

Comment:
This one would be representative in the new set, right? Want to add `  # representative: true` (commented out) at the end?

Reply:
Updated